### ISSUE TITLE
Improvement suggestion: SnackBar hidden behind open keyboard

### ISSUE THREAD

**Issue by eduardothiesen** [Flutterando/asuka#32]
When using Asuka.showSnackBar(...), if the keyboard is open, the snackbar opens behind it, making it impossible to see the message.

Current workaround: always hide keyboard on action that can fire the snackbar.

**Comment by shobhit0888** [comment]
can i work on this

**Comment by tauamendonca** [comment]
@shobhit0888 of course, this is an open source package and we are always open to suggestions and code improvements. 

We from Flutterando are updating some other packages right now, so we are a bit swamped with work so any help is more than welcome!

**Comment by tauamendonca** [comment]
Related to issue #33
